components/status-indicator/test/status-indicator.visual-diff.js

```diff
@@ -0,0 +1,42 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-status-indicator', function() {
+
+	const visualDiff = new VisualDiff('status-indicator', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/status-indicator/test/status-indicator.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
+	});
+
+	after(() => browser.close());
+
+	[
+		{ title: 'no-state-subtle', fixture: '#no-state-subtle' },
+		{ title: 'default-subtle', fixture: '#default-subtle' },
+		{ title: 'success-subtle', fixture: '#success-subtle' },
+		{ title: 'alert-subtle', fixture: '#alert-subtle' },
+		{ title: 'none-subtle', fixture: '#none-subtle' },
+		{ title: 'no-state-bold', fixture: '#no-state-bold' },
+		{ title: 'default-bold', fixture: '#default-bold' },
+		{ title: 'success-bold', fixture: '#success-bold' },
+		{ title: 'alert-bold', fixture: '#alert-bold' },
+		{ title: 'none-bold', fixture: '#none-bold' },
```

Comment:
Nit: looks like the input here could be simplified since the ID is just the title prepended by a `#` in all cases.